TACC-Cloud/hazmapper
# Task/wg 381 assets view image video no asset

## Overview: ##
Create asset detail panel similar to current prod angular detail panel
## PR Status: ##


## Related Jira tickets: ##

* [WG-381](https://tacc-main.atlassian.net/browse/WG-381)

## Summary of Changes: ##

- Adds place and styles for panel on MapProject page
- Changes useCurrentFeature hook so that it picks up the latest query accurately
- Creates asset detail component with styles
- Creates asset detail placeholders for questionnaire and pointcloud
- Need to add meaningful test

## Testing Steps: ##
1. Click a map and click around various assets with the page and make sure they appear in the display panel
2. Check video, image, point, and geometry type assets to make sure they display as expected
3. Click the same highlighted asset and make sure the panel and search param disappears
4. Go to different map and make sure it displays as expected.

NOTES: 

- Questionnaires and pointclouds are a follow on. 
- Add an asset for a feature with no asset button currently doesn't work. 
- Download button currently doesn't work


## UI Photos:
### Movies
<img width="1095" alt="Screenshot 2024-12-12 at 4 49 00 PM" src="https://github.com/user-attachments/assets/e1d3a13b-fd01-41b6-b2aa-e0f06fb7418f" />

###Images
<img width="1106" alt="Screenshot 2024-12-12 at 4 49 52 PM" src="https://github.com/user-attachments/assets/e0d3153a-2d78-4445-bd50-1bcef6d3a45f" />

Points:

<img width="1096" alt="Screenshot 2024-12-12 at 4 50 37 PM" src="https://github.com/user-attachments/assets/dcc6bbb4-deaf-4460-a834-70c387ca5bb1" />


## Notes: ##
